test/development/basic/styled-components-disabled/pages/index.js

```diff
@@ -13,7 +13,7 @@ const Button = styled.a`
   /* The GitHub button is a primary button
    * edit this to target it specifically! */
   ${(props) =>
-    props.primary &&
+    props.$primary &&
```

Comment:
Drive-by to avoid prop forwarding to host elements which than trigger "unrecognized prop on DOM element [...]" warnings.